src/storage.js

```diff
@@ -6,6 +6,15 @@ const _ = require('lodash');
 module.exports = function(storage, progress) {
   return storage.read()
     .then(function(data) {
+      progress.rules = _.map(progress.rules || [], function(rule) {
```

Comment:
We've been fighting this issue in a number of way PR #12 I introduced a property filter on these large files.  Once the propertyReduce removed the properties they were replaced with a checksum of the file.  This might be worth doing for consistency.  

WDYT?

Besides this the code seems fine.  Let me know if you think it is worth doing.